autotests/helm/values.yaml

```diff
@@ -25,8 +25,8 @@ global:
     activeDeadlineSeconds: 3600 # 1h
 
   env:
-    PARTICIPANT_NAME: <REPLACE_WITH_USERNAME>
-    api_host: http://inca-smc-mlops-challenge-solution.default.svc.cluster.local/<REPLACE_WITH_ENDPOINT>
+    PARTICIPANT_NAME: Khush Patel
+    api_host: http://inca-smc-mlops-challenge-solution.default.svc.cluster.local/8000
```

Comment:
I assume /process is your endpoint, port mapping is configured in solution/helm/env/